__tests__/port.test.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const Port = require('../src/port.js');
+
+describe('Port', () => {
+    let southamptonConfig;
+    let southampton;
+    beforeEach(() => {
+        southamptonConfig = {
+            name: jest.fn(),
+            _ships: []
+        };
+        southampton = new Port(southamptonConfig);
+    });
+
+    describe('constructor', () => {
+        test('Port can be instantiated', () => {
+            expect(southampton).toBeInstanceOf(Port);
+        });
+
+        test('Port instances have own "name" property', () => {
+            expect(southampton).toHaveProperty('name');
+        });
+
+        test('Port instances have own "_ships" property', () => {
+            expect(southampton).toHaveProperty('_ships');
+        });
+    });
+
+    describe('addShip', () => {
+        test('State of "_ships" should change depending on docked ships', () => {
+            const ship1 = jest.fn();
+            const ship2 = jest.fn();
+
+            southampton.addShip(ship1);
+            expect(southampton._ships).toEqual([ship1]);
+            
+            southampton.addShip(ship2);
+            expect(southampton._ships).toEqual([ship1, ship2]);
```

Comment:
Assertions can all be grouped together at the end for easier readability
```suggestion
            southampton.addShip(ship1);
            southampton.addShip(ship2);

            expect(southampton._ships).toEqual([ship1]);
            expect(southampton._ships).toEqual([ship1, ship2]);
```

Reply:
Would this work? I'll have to try this but I would have thought the first expect would always equal `[ship1, ship2]` if I called `addShip` twice before the first expect?

Reply:
You may be right there, I didnt test this. If if doesnt work please disregard